.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "Ubuntu",
```

Comment:
tabs -> spaces

Reply:
hah i didn't even notice that the default config it generated used tabs. fixed.